tests/test_docopt.py

```diff
@@ -708,22 +715,226 @@ def test_issue_71_double_dash_is_not_a_valid_option_argument():
 usage: pit stop"""
 
 
-def test_parse_section():
-    assert parse_section("usage:", "foo bar fizz buzz") == []
-    assert parse_section("usage:", "usage: prog") == ["usage: prog"]
-    assert parse_section("usage:", "usage: -x\n -y") == ["usage: -x\n -y"]
-    assert parse_section("usage:", usage) == [
-        "usage: this",
-        "usage:hai",
-        "usage: this that",
-        "usage: foo\n       bar",
-        "PROGRAM USAGE:\n foo\n bar",
-        "usage:\n\ttoo\n\ttar",
-        "Usage: eggs spam",
-        "usage: pit stop",
-    ]
-
-
 def test_issue_126_defaults_not_parsed_correctly_when_tabs():
     section = "Options:\n\t--foo=<arg>  [default: bar]"
-    assert parse_defaults(section) == [Option(None, "--foo", 1, "bar")]
+    assert parse_options(section) == [Option(None, "--foo", 1, "bar")]
```

Comment:
How about we just add this as a test case to the cases below? Before, we didn't have this "unit" testing for parsing options, so it made sense to have this as a separate test. But now we do have very similar tests in `test_parse_options`, so just put this case in there too. Having it separate implies it is somehow special, but I think this is just another instance of those other cases we test.

Reply:
- [x] remove redundant test `test_issue_126_defaults_not_parsed_correctly_when_tabs` (include its example in main options test)